### DISCUSSION TITLE
Scenicplus for comparative analysis between species or conditions?

### DISCUSSION THREAD

**Discussion by jding5066** [aertslab/scenicplus#236] [Q&A]
Hello,

Thank you so much for the great tool! 

I was wondering what would be the best way to compare GRNs between species aligned to different genomes and if there are any functions in scenicplus that support differential GRN testing between samples.

Thank you in advance for your help!

**Comment by SeppeDeWinter** [comment]
Hi @jding5066 

There are no particular functions in the SCENIC+ repo to do these types of analysis. 
However, for inspiration you could have a look at the SCENIC+ publiction. Here, we compared GRNs from human and mouse brain.

https://www.nature.com/articles/s41592-023-01938-4#Sec7 (figure 5).

I hope this helps a bit?

Best,

Seppe

**Reply by jding5066** [reply]
Hi Seppe,

Thanks for the guidance! I have another question in terms of running sceniplus with custom genome.
 
I've been trying to run scenicplus with chimpanzee samples. I created a pycistarget databse based on the consensus peak called in my sample, and used the human motif2TF file for pycistopic. 

For the motif enrichment output I got a lot of unannotated motifs, which I did not see on the paired data from human samples.  
<img width="856" alt="Screen Shot 2023-10-05 at 6 01 09 PM" src="https://github.com/aertslab/scenicplus/assets/91573218/23ba10cf-83b4-4180-a2c5-ccdfb2d2d358">

Also, after I finished the scenicplus pipeline, there were very few regulons stored in the scenicplus object (less than 20). The TF list I used included more than 1000 TFs detected in the chimp RNA data. 
![image](https://github.com/aertslab/scenicplus/assets/91573218/4495ee87-805e-4204-a663-be6fe3f2d062)

Do you know what might have caused this or what should I do to improve it?

Thank you!
